core/src/main/java/bio/terra/pearl/core/dao/survey/AnswerDao.java

```diff
@@ -79,4 +79,24 @@ public Optional<Answer> findForEnrolleeByQuestion(UUID enrolleeID, String survey
                         .findFirst()
         );
     }
+
+    public Optional<Answer> findByProfileIdByStudyAndQuestion(UUID profileId, String studyName, String surveyStableId, String questionStableId) {
```

Comment:
this is a tough one to name...  but the two 'bys' threw me off

```suggestion
    public Optional<Answer> findByProfileIdStudyAndQuestion(UUID profileId, String studyName, String surveyStableId, String questionStableId) {
```